dts/bindings/display/lcd-interface.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 NXP
+# Common fields for LCD interface based display controllers
+
+include: display-controller.yaml
+
+properties:
+  pixel-clock:
+    type: int
+    description: |
+      Pixel clock for display controller in Hz. Must be at least as large as:
+      (display height + VSYNC pulse width + vertical front porch + vertical back porch) *
+      (display width + HSYNC pulse width + horizontal front porch + horizontal back porch) *
+      desired frame rate
+
+  hsync:
+    type: int
+    required: true
+    description: Horizontal synchronization pulse duration, in pixels
+
+  vsync:
+    type: int
+    required: true
+    description: Vertical synchronization pulse duration, in lines
+
+  hbp:
+    type: int
+    required: true
+    description: Horizontal back-porch duration, in pixels
```

Comment:
let's align here with `panel-*` bindings, e.g. https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/display/panel/panel-timing.yaml

"panel" name is also more generic (LCD is actually a technology)

Reply:
No problem with aligning to Linux here- I'm going to wait to update this until we resolve the discussions with the MIPI binding.